bestfirstsearch.java

```diff
@@ -104,13 +109,16 @@ public void bestfirstsearch()
 visited[start_vertex]=1;
 
 int arr[]=new int[v];
-int j=0;
+@LTLengthOf("arr") int j=0;
 while(!pg.isEmpty())
 {
 pair p1=pg.remove();
 int node=p1.node;
 arr[j++]=node;
+	
+@SuppressWarnings("unused")
```

Comment:
Suppressing warnings that are not Checker Framework issued is not relevant for the case study.

Reply:
I have removed this line.